product_docs/docs/pgbouncer/1.16/pgbouncer_rel_notes/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Release Notes"
+redirects:
+  - ../01_whats_new/
+---
+
+The Pgbouncer documentation describes the latest version of Pgbouncer 1.16.0.1 including minor releases and patches. The release notes in this section provide information on what was new in each release. For new functionality introduced in a minor or patch release, there are also indicators within the content about what release introduced the feature.
```

Comment:
```suggestion
The Pgbouncer documentation describes the latest version of Pgbouncer 1.16 including minor releases and patches. The release notes in this section provide information on what was new in each release. For new functionality introduced in a minor or patch release, there are also indicators within the content about what release introduced the feature.
```
No need to give the patch number since we have this wording following the version number: "including minor releases and patches>"